src/functions-and-arrays.js

```diff
@@ -1,19 +1,49 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(a, b) {
+  if (a>b){
+    return a
+  }
+  else if (b>a) {
+    return b
+  }   
+  else if (a=b) {
```

Comment:
Instead of your else if-statements, you could just write `return b`, because if a ist not the highest number, b is the highest number. If both are equal, it doesn't matter which you return.